### PR TITLE
update phpunit version

### PR DESCRIPTION
**Pull request by gharlan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * If you are adding functionality or fixing a bug - add a test!
  * Fix project itself: `php php-cs-fixer fix`.
  * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify `README.rst` manually!
- * Check if tests pass: `phpunit` [(4.*)](https://phpunit.de/manual/current/en/installation.html)
+ * Check if tests pass: `phpunit` [(5.x)](https://phpunit.de/manual/current/en/installation.html)
```
Comment (gharlan) [inline CONTRIBUTING.md]: or `>= 4.x`?

**Comment by GrahamCampbell** [comment]
This change is incorrect.

**Comment by GrahamCampbell** [comment]
PHPUnit 5 doesn't work on php 5.3/5.4/5.5.

**Comment by gharlan** [comment]
So, `>= 4.x` would be correct?

**Comment by GrahamCampbell** [comment]
> So, >= 4.x would be correct?

No.

**Comment by GrahamCampbell** [comment]
"4.x or 5.x" is correct

**Comment by GrahamCampbell** [comment]
We don't know that php 7.x or something might move to namespaces and break everything

**Comment by keradus** [comment]
for now ^4|^5 works
We already use PHPUnit 5 on Travis: https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/100858474#L164

**Comment by gharlan** [comment]
updated

**Comment by keradus** [comment]
Thank you @gharlan.

**Pull request merged**